workflow-templates/run-nextstrain-build-on-aws-batch.yaml

```diff
@@ -0,0 +1,28 @@
+name: Run Nextstrain build on AWS Batch
+
+on:
+  workflow_dispatch:
+    inputs:
+      image:
+        description: "Specific starter image to use for the build. Will override the default image for `nextstrain build`."
+        required: false
+        type: string
+
+jobs:
+  deploy-build-to-aws-batch:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: nextstrain/.github/actions/run-nextstrain-build-on-aws-batch@master
```

Comment:
_non-blocking_

Why `deploy-build-to-aws-batch` vs. `run-nextstrain-build-on-aws-batch`?

Reply:
Huh, I don't have a good answer here. Will consolidate to `run-build`